Ginger/GingerCoreNET/ActionsLib/ActSetVariableValue.cs

```diff
@@ -134,6 +134,10 @@ public override void Execute()
                 {
                     ((VariableDynamic)Var).ValueExpression = this.Value;
                 }
+                else if (Var.GetType() == typeof(VariableNumber))
```

Comment:
need to ensure the value to be set fulfill all validation e.g. max , min etc. if not need to fail the action